### PR TITLE
APIS-4825 -Wired in the Application Details page to get data from APM

### PR DESCRIPTION
**Pull request by anjumabbas5**

### CODE REVIEW

File: acceptance/acceptance/specs/APIGatekeeperBaseSpec.scala [hunk 1]
```diff
@@ -29,26 +29,50 @@ import play.api.libs.json.Json
...
   }
 
+  // def stubBlockedApplication(application: String, developers: List[User]) = {
+  //   stubFor(get(urlEqualTo("/gatekeeper/application/fa38d130-7c8e-47d8-abc0-0374c7f73217")).willReturn(aResponse().withBody(application).withStatus(OK)))
+  //   stubFor(get(urlEqualTo("/application/fa38d130-7c8e-47d8-abc0-0374c7f73217")).willReturn(aResponse().withBody(application).withStatus(OK)))
+  //   stubFor(get(urlEqualTo("/gatekeeper/application/fa38d130-7c8e-47d8-abc0-0374c7f73217/subscription")).willReturn(aResponse().withBody("[]").withStatus(OK)))
+  //   stubFor(get(urlEqualTo("/application/fa38d130-7c8e-47d8-abc0-0374c7f73217/subscription")).willReturn(aResponse().withBody("[]").withStatus(OK)))
+  // }
+
   def stubApplicationList() = {
```
Comment (adampridmore) [inline acceptance/acceptance/specs/APIGatekeeperBaseSpec.scala:71]: Commented out code! 😮

Comment (peteslater-ee) [reply acceptance/acceptance/specs/APIGatekeeperBaseSpec.scala:71]: For use later

Comment (peteslater-ee) [reply acceptance/acceptance/specs/APIGatekeeperBaseSpec.scala:71]: Actually it's not for use later - it's wrong

File: app/connectors/ApmConnector.scala [hunk 2]
```diff
@@ -17,39 +17,37 @@
...
-  //   import domain.services.ApplicationsJsonFormatters._
-  //   import domain.services.SubscriptionsJsonFormatters._
+   def fetchApplicationById(applicationId: ApplicationId)(implicit hc: HeaderCarrier): Future[Option[ApplicationWithSubscriptionData]] =
+     http.GET[Option[ApplicationWithSubscriptionData]](s"${config.serviceBaseUrl}/applications/${applicationId.value}")
 
-  //   http.GET[Map[ApiContext, Map[ApiVersion, Map[FieldName, SubscriptionFieldDefinition]]]](s"${config.serviceBaseUrl}/subscription-fields?environment=$environment")
-  // }
+  //  def getAllFieldDefinitions(environment: Environment)(implicit hc: HeaderCarrier): Future[Map[ApiContext,Map[ApiVersion, Map[FieldName, SubscriptionFieldDefinition]]]] = {
+  //    http.GET[Map[ApiContext, Map[ApiVersion, Map[FieldName, SubscriptionFieldDefinition]]]](s"${config.serviceBaseUrl}/subscription-fields?environment=$environment")
+  //  }
```
Comment (adampridmore) [inline app/connectors/ApmConnector.scala:43]: Comment out cod!

Comment (peteslater-ee) [reply app/connectors/ApmConnector.scala:43]: That's for use later

Comment (peteslater-ee) [reply app/connectors/ApmConnector.scala:43]: For real this time

File: app/model/Application.scala [hunk 3]
```diff
@@ -208,7 +218,7 @@ object ApplicationResponse {
     .and[Privileged](AccessType.PRIVILEGED.toString)
     .and[Ropc](AccessType.ROPC.toString)
     .format
-  implicit val format1 = Json.format[APIIdentifier]
+  // implicit val format1 = Json.format[APIIdentifier]
   implicit val formatRole = EnumJson.enumFormat(CollaboratorRole)
```
Comment (adampridmore) [inline app/model/Application.scala:222]: Commenty!

**Review by adampridmore — APPROVED** [review]

**Pull request merged**